tests/Integration/HTTP/ClientsTest.php

```diff
@@ -43,7 +43,7 @@ public function testPrs18ClientGetContentOfLocalFile(): void
 
     private function runTestsWithClientGetContentOfLocalFile(Client $client): void
     {
-        $filepath = dirname(__FILE__, 3) . '/data/feed_rss-2.0.xml';
+        $filepath = dirname(__FILE__, 3).'/data/feed_rss-2.0.xml';
```

Comment:
Is this whitespace change on purpose?

Reply:
Oops, accidentally committed Symfony CS changes.